rust-runtime/aws-smithy-http-server/Cargo.toml

```diff
@@ -41,6 +41,7 @@ tracing = "0.1.35"
 tokio = { version = "1.8.4", features = ["full"] }
 tower = { version = "0.4.11", features = ["util", "make"], default-features = false }
 tower-http = { version = "0.3", features = ["add-extension", "map-response-body"] }
+uuid = { version = "1", features = ["v4", "fast-rng", "macro-diagnostics"] }
```

Comment:
The server request id stuff should be behind a feature flag (which we can enable by default), since it brings in extra deps.